recipes/pyhacrf-datamade/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyhacrf-datamade" %}
+{% set version = "0.2.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyhacrf-datamade-{{ version }}.tar.gz
+  sha256: cf9dc239090696cee301d5d32074e1ed881853c6da789ce05817c072b22c4f44
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python 
+    - pylbfgs
```

Comment:
I don't think this is needed at build-time?
```suggestion
```

Reply:
Okay does that mean I should include the windows build afterall?

Reply:
No, because it's still needed to run the package. It's just not needed while building the package.